.unreleased/LLT-5252

```diff
@@ -0,0 +1 @@
+ci: Update gitlab urls
```

Comment:
I guess when it is not a PR introducing sth used by apps this file should be empty

Reply:
How can I tell that to `check-changelog` job? 👀